# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/w3m/no_egd.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- w3m-0.5.3/url.c.vanilla	2021-01-22 15:06:49.816000000 +0100
+++ w3m-0.5.3/url.c	2021-01-22 15:07:22.325000000 +0100
@@ -268,6 +268,8 @@
     if (RAND_status())
 	return;
     if ((file = RAND_file_name(buffer, sizeof(buffer)))) {
+#undef USE_EGD
+/* Note: the alterative would be to enable building of EGD in openssl, I suppose */
 #ifdef USE_EGD
 	if (RAND_egd(file) > 0)
 	    return;
